packages/db-objects-redis/src/lib/objects/objectsUtils.ts

```diff
@@ -329,41 +333,42 @@ export function checkFileRights(
      return callback(null, options);*/
 }
 // For users and groups
-function getDefaultAdminRights(acl?: Record<string, any>, _isState?: boolean) {
-    acl = acl || {};
-    acl.file = {
-        list: true,
-        read: true,
-        write: true,
-        create: true,
-        delete: true
-    };
-    acl.object = {
-        create: true,
-        list: true,
-        read: true,
-        write: true,
-        delete: true
-    };
-    acl.users = {
-        create: true,
-        list: true,
-        read: true,
-        write: true,
-        delete: true
-    };
-    acl.state = {
-        read: true,
-        write: true,
-        delete: true,
-        create: true,
-        list: true
+function getDefaultAdminRights(acl?: ioBroker.ObjectPermissions, _isState?: boolean) {
```

Comment:
All the existing structures for permissions/ACL/etc. are pretty messy. I did capture most of them here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20f9baa9f3dbf1eb5cfc5c3c7d974b92b211fc61/types/iobroker/index.d.ts#L68-L104
but I believe there are other variants in use too.

Reply:
Not sure if `ObjectPermissions` is actually the intended type here.